### PR TITLE
ADEN-6531 Disable jwplayer tag ads when UAP is loaded

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/articleVideoAd.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+		featuredVideoSlotName = 'FEATURED',
+		logGroup = 'ext.wikia.adEngine.video.articleVideoAd';
+
+	function calculateRV(depth) {
+		var capping = adContext.get('opts.fvAdsFrequency');
+
+		return (depth < 2 || !capping) ? 1 : (Math.floor((depth - 1) / capping) + 1);
+	}
+
+	function buildVastUrl(position, videoDepth, correlator, bidParams) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/video/articleVideoAd.js]: What happened with slotTargeting? :)

File: extensions/wikia/AdEngine/js/provider/btfBlocker.js [hunk 2]
```diff
@@ -24,11 +24,12 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 	function isBTFDisabledByCreative() {
 		return win.ads.runtime.disableBtf;
 	}
+	var btfQueue = [],
+		btfQueueStarted = false,
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/provider/btfBlocker.js:28]: Should we clear this state on consecutive pvs on mobile-wiki?

File: extensions/wikia/JWPlayerTag/scripts/jwplayertag.ads.js [hunk 3]
```diff
@@ -69,7 +70,11 @@ define('wikia.articleVideo.jwplayertag.ads', [
 				videoDepth += 1;
 
 				trackingParams.adProduct = 'video-preroll';
-				player.playAd(buildVastUrl('preroll', videoDepth, correlator, slotTargeting));
+
+				btfBlocker.decorate(function() {
+					player.playAd(buildVastUrl('preroll', videoDepth, correlator, slotTargeting));
```
Comment (fraszczakszymon) [inline extensions/wikia/JWPlayerTag/scripts/jwplayertag.ads.js]: Can we reuse `ext.wikia.adEngine.video.articleVideoAd` here?

File: extensions/wikia/AdEngine/js/provider/btfBlocker.js (hunk 2 above)
Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/provider/btfBlocker.js:28]: I believe we already do it: https://github.com/Wikia/app/pull/14650/files#diff-3ede2fc47081feff0bafc6b748c2cf03R37

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**